First configuration table (#14)
SW814F20/Report: report/Sprint_1/ConfigurationTable.tex

\section{Configuration Table}\label{sec:first-configuration-table}
This section will describe what a configuration table is, and how our configuration table looks for the first sprint.

A configuration table, is a table, with information about the project status. 

> The table will help you get a better overview of the solution you are trying to create, and why the problem needs to be solved.

Reviewer: ```suggestion
The table will help get a better overview of the solution being made, and why the problem needs to be solved.
```

\section{Configuration Table}\label{sec:first-configuration-table}
This section will describe what a configuration table is, and how our configuration table looks for the first sprint.

> So what is a configuration table, you might ask?

Reviewer: ```suggestion
```

Author: Ups. Troede jeg havde fjernet den :D

A configuration table, is a table, with information about the project status. 
The table will help you get a better overview of the solution you are trying to create, and why the problem needs to be solved.

> Furthermore, you can at all times see what the given sprint focus is, and when you need to review the solution and other things, then you can just look at the configuration table, as one of the columns will be the criteria for the solution.

Reviewer: ```suggestion
Furthermore, at all times it is visible what the given sprint focus is, and when review of the solution and other things is needed, the configuration table has columns with criteria for the solution.
```

\begin{figure}[h]
    \centering
    \includegraphics[width=\linewidth]{images/configurationTableExample.png}
    \caption{Structure of a Configuration table.}
    \label{fig:example-configuration-table}
\end{figure}

~\autoref{fig:example-configuration-table} shows the stucture of a configuration table.
Here we have three areas, \textbf{Rationale}, \textbf{Tactics} and \textbf{Strategy}.
Each of these has its own area of view in the given project, however, there are some categories which they share with one another.
Furthermore, the configuration table is organised into four "views", which are:

\begin{itemize}
    \item Paradigm - Used as the understanding of the problem
    \item Product - Used to describe how the constructive parts of the solution could be designed
    \item Project - Used to reason about the purpose and our overall idea in the project.
    \item Process - Used to determine if we are on the right track or not, in solving the problem. 
\end{itemize}

> Furthermore they all involves the \textbf{Paradigm}, which is the reflection column, the \textbf{Product}, which is 

Reviewer: Der bruges meget "Furthermore"
Jeg tror vi skal have noget variation i sproget der bliver brugt

> 

Reviewer: Hvorfor er der linebreak her?

The rationale layer is known as the "why" layer.

> Why are we trying to solve the problem we have, and which leverage points do we use in order to solve the problem.

Reviewer: ```suggestion
Why are we trying to solve the problem we have, and which leverage points is used to solve the problem.
```

Under paradigm, we have \textbf{Challenge} and \textbf{Problem}, also known in the first iteration, as the initial problem.
In the product column, we have \textbf{Leverage}, which is the leverage points we will use in this iteration, to solve the problem we have.
Next, we have the resolution, which is the prospect, warrant, and backing. 
Here we look at the \textbf{Prospect}, which is the solution, we believe will solve the problem.
The \textbf{Warrant} is a statement about why it is important to solve the problem at hand, and the \textbf{Backing} is why we think that the prospect is a useful solution to the problem.
The last cell is the rationale review, where we will look at the \textbf{Criteria for our resolution}, which involves our Prospect, warrant, and backing.
This cell is meant to be a checklist, which can be reviewed, for example at the end of a sprint.

> The rationale layer also involves the architecture, which we will describe in the strategy layer paragraph.

Reviewer: ```suggestion
The rationale layer also involves architecture, which will be described in the strategy layer paragraph.
```
